### ISSUE TITLE
Black textures 

### ISSUE THREAD

**Issue by f00god** [ZenoArrows/The-Simpsons-Hit-and-Run#6]
Sometimes during the game some textures turn black and the game starts stuttering a lot.

**Comment by ZenoArrows** [comment]
If you can find any steps to reproduce it that would be very helpful in tracking down the issue.

**Comment by f00god** [comment]
Might be like that: 
1. Play 4-5 missions on 2nd level.
2. Finish 2nd level and continue playing on 3rd level.
3. Play 4-5 missions on 3rd level.
4. Some textures become black, the more you play like this, the more textures become black.
The FPS has dropped a bit, but the more you play, the more new black textures you see, FPS decrease more and more until the "slideshow" appears.

**Comment by ZenoArrows** [comment]
Sounds like there may be a memory leak after playing for a long time. I assume it works fine after restarting the game?

**Comment by f00god** [comment]
Yes. Works fine.

**Comment by ZenoArrows** [comment]
Can you still reproduce this in version 0.4?

**Comment by f00god** [comment]
Yes, it's still possible. I restart the game every time I switch levels. 
![2023102901385300-B662483B2A4D01561FA37775965F956C](https://github.com/ZenoArrows/The-Simpsons-Hit-and-Run/assets/147378676/3faf7cd3-c907-447b-844c-e8883f16c4cc)
![2023102900581300-B662483B2A4D01561FA37775965F956C](https://github.com/ZenoArrows/The-Simpsons-Hit-and-Run/assets/147378676/41b3efbb-148c-4503-9581-e3f0941b940c)
